src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -2920,6 +2928,37 @@ function doSpeedWalk()
     end
 end
 
+function map.pauseSpeedwalk()
+    if #speedWalkDir ~= 0 then
+        map.echo("Speedwalking paused.")
+        walking = false
+        raiseEvent("sysSpeedwalkPaused")
+    else
+        map.echo("No active speedwalk found.")
+    end
+end
+
+function map.resumeSpeedwalk(delay)
+    if #speedWalkDir ~= 0 then
+        map.echo("Speedwalking resumed.")
+        map.find_me(nil, nil, nil, true)
+        raiseEvent("sysSpeedwalkResumed")
+        tempTimer(delay or 0, function() map.speedwalk(nil, speedWalkPath, speedWalkDir) end)
+    else
+        map.echo("No active speedwalk found.")
```

Comment:
```suggestion
        map.echo("Not currently speedwalking.")
```

Reply:
ok， Let me change it，I wrote these prompts from the function prompts in the API